shell/platform/darwin/ios/framework/Source/FlutterViewControllerTest.mm

```diff
@@ -1851,6 +1851,8 @@ - (void)testMouseSupport API_AVAILABLE(ios(13.4)) {
 
   [vc discreteScrollEvent:mockPanGestureRecognizer];
 
+  // The mouse position within panGestureRecognizer should be checked
```

Comment:
Is it testable to check if the mouse location was also updated, with the right values?

Reply:
I couldn't figure it out. because this is a primitive value / non-Object, OCMock stub-and-check-argument can't be used.